R/cand_sector.R

```diff
@@ -26,14 +29,50 @@ cand_sector <- function(candidate_id, cycle = 2018, api_key = get_os_key()) {
 		httr::stop_for_status() %>% 
     httr::content("text") %>%
     jsonlite::fromJSON()
+# NEXT
+  process_cand_sector(res)
+
+
+  # REMOVE
+  # res%>%
+  #   pluck("response", "sectors", "sector", "@attributes") %>%
+  #   mutate(candidate = cand_name,
+  #          party = cand_party,
+  #          cycle = res$response$sectors$`@attributes`$cycle,
+  #          cid = res$response$sectors$`@attributes`$cid)%>%
+  #   select(candidate, party, cycle, everything()) %>%
+  #   as_tibble()
+
+}
+
+#' @title process_cand_sector
+#' @description process res that returned from API CALL
+#'
+#'
+#'
+#'
+  process_cand_sector  <- function(res = NULL){
```

Comment:
I do not think that there needs to be a separate function for the purpose of processing the response. Can you move this within the function itself?